### ISSUE TITLE
Method for removing user backgrounds

### ISSUE THREAD

**Issue by MattTucker22689** [jneilliii/OctoPrint-CustomBackground#26]
Could use a method for removing the user's background. 

Also, a method to change the color scheme, and transparency of an image would be nice. I'm going to look into it and see if I can figure out a method for adjusting those via a slider and an extra transparent window over top the image to tint it with.

**Comment by jneilliii** [comment]
Good point.  At this time, you can't remove the image directly, but you can edit config.yaml and remove the settings and then the default image will be restored.

As for the other options, if you figure it out, I'd be happy to merge in the changes for a future version.
